packages/examples/resolver-diagnostics/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@parcel/resolver-diagnostics-example",
+  "version": "2.0.0-alpha.3.2",
+  "license": "MIT",
+  "private": true,
+  "scripts": {
+    "demo": "parcel build src/index.js --no-cache"
```

Comment:
For some reason this is not bundling node_modules?

Reply:
I'm guessing due to Devon's PR to resolve targets, the browserslist isn't used at all (the root package.json has `engines.node`)

Reply:
Can I fix it somehow?

Reply:
add an empty `packages/examples/resolver-diaggnostics/yarn.lock`

Reply:
That kinda sucks maybe we should fix that? Seems like it would come up a lot in monorepos. Doesn't seem unlikely to have a front-end and api in one monorepo

Reply:
The empty yarn.lock works awesome, thanks

Reply:
Indeed, https://github.com/parcel-bundler/parcel/pull/4098 broke many tests and also the benchmark action.